django/conf/urls/__init__.py

```diff
@@ -17,6 +16,8 @@
 
 
 def include(arg, namespace=None, app_name=None):
```

Comment:
Is there a reason to keep the URL-related APIs split between `django.conf.urls` and `django.urls`?

I believe everything should be moved to `django.urls`.